### PR TITLE
Feature/pill component

### PR DESCRIPTION
**Pull request by shaheenhad**
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] If there are changes to components, I have added / updated automated tests
- [x] I have made any necessary updates to Storybook to accompany these changes
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/theme/custom.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@ $white15: rgba(255, 255, 255, 0.15);
 $white20: rgba(255, 255, 255, 0.2);
 $white35: rgba(255, 255, 255, 0.35);
 $white60: rgba(255, 255, 255, 0.6);
+$white70: rgba(255, 255, 255, 0.7);
```
Comment (jahorwitz) [inline src/theme/custom.scss:9]: With the updated Opacities, We should be able to delete all of the `$whiteXX` and `$blackXX` from our theme. Can you search for anywhere we are using those values and update them accordingly?

**Review by jahorwitz — CHANGES_REQUESTED** [review]
Just a few changes 👍

File: src/theme/custom.scss (hunk 1 above)
Comment (shaheenhad) [reply src/theme/custom.scss:9]: So, I added this as the combination of the `text-white70` and `text-opacity-70` was the only way I could get the effect to work, same for the `bg-opacity`.

File: src/components/pill/pill.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * Pill Component
+ * @param shape expects 'round' or 'square'
+ * @param isCopiable expects 'true' or 'false'
+ * @param text expects the text to be displayed in the pill
+ * @param opacity expects the pill background opacity '10' or '20'
+ * @param textOpacity expects the text opacity '100' or '70'
+ * @param fs expects the font size
+ */
+
+const CopyBtn = styled.button`
```
[the remark this answers is not in the document]
Comment (shaheenhad) [reply src/components/pill/pill.jsx:15]: I did not see any other states for this in figma, but will add the TODO.

File: src/components/pill/pill.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  border: none;
+  background-color: transparent;
+`;
+
+export const Pill = ({ isCopiable, text, shape, opacity, textOpacity, fs }) => {
+  const [pillText, setPillText] = useState("");
```
[the remark this answers is not in the document]
Comment (shaheenhad) [reply src/components/pill/pill.jsx:26]: overkill on my part, corrected.

**Pull request merged**